src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -159,6 +159,19 @@ private CentralPackageVersion ToCentralPackageVersion(string packageId, string v
             return new CentralPackageVersion(packageId, VersionRange.Parse(version));
         }
 
+        private IReadOnlyDictionary<string, PrunePackageReference> GetPackagesToPrune()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            IEnumerable<(string PackageId, string Version)> packageVersions =
+                        _vsProjectAdapter.GetBuildItemInformation(ProjectItems.PrunePackageReference, ProjectBuildProperties.Version)
+                        .Select(item => (PackageId: item.ItemId, Version: item.ItemMetadata.FirstOrDefault()));
+
+            return packageVersions
+                .Select(item => PrunePackageReference.Create(item.PackageId, item.Version))
+                .ToDictionary(cpv => cpv.Name, StringComparer.OrdinalIgnoreCase);
```

Comment:
```suggestion
                .ToDictionary(i => i.Name, StringComparer.OrdinalIgnoreCase);
```

Looks like this was copy/pasted from `CentralPackageVersions`...

Reply:
Yep it was :D 
I need the same method calls D: